### PR TITLE
adds support and tests for interleaved input files

### PR DESCRIPTION
**Pull request by bwlang**

### CODE REVIEW

**Comment by bwlang** [comment]
any concerns about the new dependency on seqtk or the somewhat hacky way i'm supporting bzip2?

**Comment by bgruening** [comment]
@galaxyproject/iuc this is an invasive change to Hisat, can you please comment and review :)

Thanks!

File: tools/hisat2/hisat2.xml [hunk 1]
```diff
@@ -155,10 +176,14 @@ hisat2
     #end if
 
 #else:
-
-    -1 '${read1}'
-    -2 '${read2}'
-
+    ##quotes are embedded in r1 and r2 variables, needed to allow use of <()
```
Comment (mblue9) [inline tools/hisat2/hisat2.xml]: I don't know enough to know if this is ok or if there's a better way to do this, otherwise this PR is fine by me!

**Comment by bwlang** [comment]
looks good to me - thanks for the fixes.

**Comment by abretaud** [comment]
Thanks @bwlang!

File: tools/hisat2/hisat2.xml (hunk 1 above)
Comment (jmchilton) [reply tools/hisat2/hisat2.xml]: This isn't super clean but looks entirely reasonable to me and I can't imagine a trick to clean it up any further. In this case ``${read1}`` doesn't have any unwanted characters in it as far as I can tell based on the above logic - so this seems secure enough to me.

Comment (mblue9) [reply tools/hisat2/hisat2.xml]: Cool, good to know, thanks @jmchilton !

**Pull request merged**